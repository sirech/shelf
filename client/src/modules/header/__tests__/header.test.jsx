import React from 'react'
import { shallow } from 'enzyme'
import Header from '../header'

const header = () => {
  return shallow(<Header />)
}

describe('components', () => {
  describe('Header', () => {
    it('renders', () => {
      expect(header().length).toEqual(1)
    })

    it('links correctly', () => {
      const links = header().find('Link').map(node => node.props().to)
      expect(links).toEqual(['/', '/books', '/games'])
    })
  })
})
