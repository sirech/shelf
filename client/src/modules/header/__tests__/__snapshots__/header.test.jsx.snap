exports[`components Header renders correctly 1`] = `
<header>
  <nav
    className="navbar navbar-fixed-top navbar-inverse bg-inverse navbar-toggleable-md">
    <div
      className="navbar-collapse">
      <ul
        className="navbar-nav mr-auto">
        <li
          className="navbar-item">
          <a
            className="navbar-brand"
            onClick={[Function]}
            style={Object {}}>
            Shelf
          </a>
        </li>
        <li
          className="nav-item ml-2">
          <a
            className="nav-link"
            onClick={[Function]}
            style={Object {}}>
            Books
          </a>
        </li>
        <li
          className="nav-item ml-2">
          <a
            className="nav-link"
            onClick={[Function]}
            style={Object {}}>
            Games
          </a>
        </li>
      </ul>
    </div>
  </nav>
</header>
`;
